math/libtommath/Portfile

```diff
@@ -36,6 +36,7 @@ use_configure       no
 variant universal {}
 
 build.args          LIBPATH=${prefix}/lib \
+                    V=1 \
```

Comment:
“Disable silent rules”, i.e. show the complete command for each step during the build rather than some abbreviated message. As done in many other ports (usually during configure instead); useful for debugging. I have added a comment for this.